### ISSUE TITLE
[Feature Request] Stop pinging darkreader website setting

### ISSUE THREAD

**Issue by jazhanford** [darkreader/darkreader#6141]
# Feature Request

## Is your feature request related to a problem? Please describe

The extension pings the darkreader website on a few occasions like new install/uninstall, extension popup open blog updates. I remember @Gusted said before he will provide a setting to turn off all website pings but that is 2 years ago now. Please add this setting to turn off all pings!

## Describe the solution you'd like

Add a setting so any and all website pings can be disabled completely

**Comment by Gusted** [comment]
>  @Gusted said before he will provide a setting to turn off all website pings but that is 2 years ago now.

That's a pretty bold claim, given the fact I've been around the darkreader repo a little over a year now and have been a happy user since ~1.5 years by now. I'd also would love to see the comment from 2 years ago where I said this.

The current PR #4472 adds a button to disable the ping of dark reader blog posts. And that PR was purely made because it was noticed in some other issue: https://github.com/darkreader/darkreader/issues/4468#issuecomment-742565096

Regards,
Gusted

**Comment by jazhanford** [comment]
Sorry @Gusted I thought you said it then but maybe my memory is wrong. This past year has felt like 3 years lol.

I didn't know about that PR. Does that cover all pings then? Looks like the install/uninstall page loads won't be covered by that PR.

**Comment by Gusted** [comment]
> Does that cover all pings then? Looks like the install/uninstall page loads won't be covered by that PR.

You cannot disable the install after you installed dark reader, the uninstall one should be looked into, as that's set at the installation(as required by the stores to ensure "safety").

**Comment by jazhanford** [comment]
That's true about the install page. What about just having a link to the install page when the UI is opened for the first time so anyone who wants to visit it can click it if they want? That seems better than automatically opening the website on install.

**Comment by Gusted** [comment]
That could be done, but I'd really like to switch that when the new design is forced on all users, as the new design allows us for such "pop-up" that can show "It seems like you've opened the UI for the first time, we'd like to thank you for using Dark Reader and suggests to read the [help page](https://darkreader.org/help) to answer common questions." Which in the default design isn't possible and which also opens by default.

**Comment by Gusted** [comment]
@alexanderby The man of the UI, might want to add some input.

**Comment by bershanskiy** [comment]
@jazhanford Soon Dark Reader will have an option to turn off news fetching. See #6637 for details.
